Inverted-Index/src/Tester.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import exceptions.NotADirectoryException;
+import file_handler.DirectoryReader;
+import file_handler.Doc;
+import file_handler.FileReader;
+import file_handler.IDirectoryReader;
+import junit.framework.Assert;
+
+public class Tester {
```

Comment:
please add more tests.

Reply:
they are going to add in "test" branch after merging this pull request

Reply:
ok